package/gluon-ebtables-net-rules/files/lib/gluon/ebtables/110-net-allow-ipv6

```diff
@@ -0,0 +1,9 @@
+siteConfig = require("gluon.site_config")
+
+rule ('NET_SRC -p IPv6 --ip6-src fe80::/10 -j RETURN')
+rule ('NET_SRC -p IPv6 --ip6-dst ff00::/8 -j RETURN')
```

Comment:
Hm, no, probably not. The advantage would be to have individual counters/statistics, which you could look at at any time with ebtables, and some sort of documentation regarding what exactly needs "::".